client/src/components/ElementEditor/Element.js

```diff
@@ -43,6 +43,8 @@ const Element = (props) => {
   const [hasUnsavedChanges, setHasUnsavedChanges] = useState(false);
   const [publishBlock] = useMutation(publishBlockMutation);
 
+  const formRenderedIfNeeded = () => formHasRendered || !props.type.inlineEditable;
```

Comment:
Honestly think it's clearer to not have this new method and just update the one place it's used to check `&& props.type.inlineEditable`

Reply:
I *have* converted it from a function to a variable though. Dunno why I thought it should be a function lol